conans/client/package_layouts/package_base_layout.py

```diff
@@ -0,0 +1,19 @@
+# coding=utf-8
+
+import os
+
+from conans.model.ref import ConanFileReference
+
+
+class PackageBaseLayout(object):
```

Comment:
Thinking about the value of this class, I think I would totally skip it at the moment and use full duck typing.